middlewares/validators/tasks.js

```diff
@@ -13,7 +14,10 @@ const createTask = async (req, res, next) => {
       links: joi.array().items(joi.string()).optional(),
       startedOn: joi.number().optional(),
       endsOn: joi.number().optional(),
-      status: joi.string().required(),
+      status: joi
+        .string()
+        .valid(...TASK_STATUS_ENUM)
```

Comment:
Can we use  the `TASK_STATUS` object only and get all the values from it like 
```suggestion
        .valid(...Object.values(TASK_STATUS))
```